src/basho_bench_stats.erl

```diff
@@ -270,6 +279,15 @@ process_stats(Now, #state{stats_writer=Module}=State) ->
             ok
     end.
 
+process_global_stats(#state{stats_writer=Module}=State) ->
+    %% Iterate over all Ops and report out each set of Statistics
+    lists:foreach(fun(Op) ->
+        %% Report out the statistics captured across the whole run
+        Stats1 = folsom_metrics:get_histogram_statistics({overall_latencies, Op}),
+        Units = folsom_metrics:get_metric_value({overall_units, Op}),
+        Module:report_global_stats(Op, Stats1, Units)
+        end, State#state.ops).
```

Comment:
Also, I think you can switch `Stats1` to just `Stats` as I don't see it used elsewhere in this function.